bmu/Src/batteries.c

```diff
@@ -1547,7 +1547,9 @@ void batteryTask(void *pvParameter)
          * - TempCellMin
          * - StateBMS
          */
-        if (sendCAN_BMU_batteryStatusHV() != HAL_OK) {
+        HAL_StatusTypeDef ret = sendCAN_BMU_batteryStatusHV();
+        vTaskDelay(3); // Added to prevent CAN mailbox full
```

Comment:
again I would look into a better way to do this, don't want to block the task bc you are inherently increasing task period

Reply:
The task period would remain constant as long as the delays don’t exceed the whole task period. The v task delay until delays the task until the task period is satisfied.